scripts/start.js

```diff
@@ -1,12 +1,10 @@
-const static = require('node-static')
-const { createServer } = require('http')
+const serve = require("node-static");
+const { createServer } = require("http");
 
-const port = 8080
-const file = new static.Server('./dist')
+const port = 8080;
+const file = new serve.Server("./dist");
 
-const listener = (req, res) => req
-  .addListener('end', () => file.serve(req, res))
-  .resume()
+const listener = (req, res) =>
+  req.addListener("end", () => file.serve(req, res)).resume();
```

Comment:
This is a good example of the newline reformatting I don't endorse. The original style is full of intent.